src/inmanta/module.py

```diff
@@ -577,9 +577,26 @@ def load_plugins(self) -> None:
         """
         if not self.loaded:
             LOGGER.warning("loading plugins on project that has not been loaded completely")
+
         for module in self.modules.values():
             module.load_plugins()
```

Comment:
1. load_plugins should be modified as well I guess, to prevent modules from being loaded twice
2. The finder should be set prior to plugin loading I would assume?

Reply:
It could be a good idea to make the modules print something upon loading, and check capsys to see if too many things are printed.